src/adapters/refract/helper.coffee

```diff
@@ -0,0 +1,12 @@
+lodash = require('lodash')
+require('lodash-api-description')(lodash)
+
+fixNewLines = (str) ->
```

Comment:
So basically the function it's removing new lines, isn't it?
In that case, I'd replace the name so I do not have to dig in the code to understand that it's doing.